### PR TITLE
init entity and migration db

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/74108

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

File: src/entities/food.entity.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  @Column({ name: 'price', type: 'int', nullable: false })
+  price: number;
+
+  @Column({ name: 'quantity', type: 'int', nullable: false })
+  quantity: number;
+
+  @Column({ name: 'unit', type: 'varchar', length: 45, nullable: false })
+  unit: string;
+
+  @Column({ name: 'description', type: 'text', nullable: false })
```
Comment (huongnt-2545) [inline src/entities/food.entity.ts]: mặc định `nullable: false` nên trường nào cần giá trị null thì em hẳng bật flag này nha, đỡ rối code

Comment (namnn17b3) [reply src/entities/food.entity.ts]: ready

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**